server/index.js

```diff
@@ -64,15 +68,23 @@ class SnapdropServer {
         }
     }
 
-    _joinRoom(peer) {
+    _joinRoom(slug, peer) {
+        // Resolve roomId from slug. Otherwise create a new roomId
+        let roomId;
+        if (typeof slug !== 'undefined' && slug.length > 1) {
+            roomId = hashids.decode(slug);
+        } else {
+            roomId = process.hrtime()[1];
```

Comment:
This is not secure and has very little randomness. Either use ``process.hrtime.bigint()`` or ``crypto.randomBytes(64).toString('hex')``.